cli/src/cluster_query.rs

```diff
@@ -1342,6 +1359,32 @@ pub fn process_get_transaction_count(rpc_client: &RpcClient, _config: &CliConfig
     Ok(transaction_count.to_string())
 }
 
+// if `--additional-fee` is not specified, no comput-budget instruction will be included in
+// transaction;
+// otherwise, user specified additional-fee lamports will be used for compute-budget instruction;
+// if `0` is specified, the max-fee of one signature will be used as default.
```

Comment:
Was thinking since `0` additional-fee doesn't have much use, maybe can override that with some default value. As you rightly pointed out, it adds more confusion than function. Without it, I can just remove `build_compute_budget_instruction` all together.